pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDArrayEx.java

```diff
@@ -411,6 +411,19 @@ public NDList dropout(NDArray input, float rate, boolean training) {
         return new NDList(JniUtils.dropout((PtNDArray) input, rate, training));
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public NDList layerNorm(
+            NDArray input, Shape normalizedShape, NDArray gamma, NDArray beta, float eps) {
+
+        return new NDList(
+                JniUtils.layerNorm(
+                        (PtNDArray) input,
+                        normalizedShape,
+                        (PtNDArray) gamma,
+                        (PtNDArray) beta,
+                        eps));
+    }
```

Comment:
Add en empty line here

Reply:
I tryed to fix it ... but could be I am blind here ... I am using
`gradlew formatJava`
and
`gradlew build`